Week3/homework/step4.js

```diff
@@ -2,7 +2,10 @@
 
 // Add your code here
 
-const addSix = createBase(6);
+const addSix = function createBase(num) {
```

Comment:
The `createBase(num)` function should return a new function that uses the `num` (or whatever name you use for that argument) as the base. So:

```js
const addSix = createBase(6);
const addSeven  = createBase(7);
console.log(addSix(10)); // --> 16
console.log(addSeven(10)); // --> 17
```